### DISCUSSION TITLE
Ventura Support

### DISCUSSION THREAD

**Discussion by yodaphone** [SkyrilHD/Dell-E7250-Hackintosh#67] [General]
Hi

Thank you so much for this. This is what i use for my daily driver E7250. Do you have any plans to support Ventura? I see the note about Apple dropping support for Macbook Pro Early 2015. However, the folks over at OpenCore Legacy Patcher have been able to port it over. I have an Early 2015 MBP & i can confirm that it works flawlessly

**Comment by SkyrilHD** [comment]
Yes, I am planning to support Ventura (with quarterly updates) However, I need to resolve some bugs before I can publish the EFI.

Current bugs:
- Ventura only boots on some situations
    - Installer does not boot
    - OTA Update does not boot
    - Already installed Ventura does boot
- TBD

If I can fix these bugs, then I will release it (+ guide)

**Reply by adyrosebrigg** [reply]
@SkyrilHD I've also been passively working on building an EFI for Ventura in the past few days, with the hopes of contributing any possible success here--although I haven't made much progress as my efforts have been split with also trying to get it working on my Chromebox. Let me know if there's anything I can do to help!

**Reply by SkyrilHD** [reply]
Tbh, I need a lot of time. I'm busy with my uni and helping friends and family, so I hardly have time to continue booting Ventura.  Hopefully I can continue building the EFI this weekend.  But an update will definitely come ;P

I also need to make separate EFIs for Monterey users which I will try keep supporting as well.

**Reply by adyrosebrigg** [reply]
Oh yeah, I didn't mean to rush you or anything! But I'm looking forward to it and can help test configs later on if needed with the hardware I have (I have both Intel 7265 & Dell (Broadcom) DW1830 wireless cards, and the 720p and 1080p displays).

**Reply by SkyrilHD** [reply]
I think I can push a beta EFI tomorrow for you guys to test. Maybe my installer is broken and causing these issues.

What should work:
- Booting into pre-installed Ventura
    - all features should work as on Monterey (except GPU acceleration obviously)
- Getting Monterey to show Ventura as an OTA update for updating with no workarounds [Updating did not work for me, so take it with a grain of salt]

**Reply by yodaphone** [reply]
wow! thats great. would be more than happy to test..

**Reply by SkyrilHD** [reply]
Okay, I managed to fix most of the issues and Ventura should be bootable. I tried booting a recovery stick and a pre-installed copy of Ventura, and both work just fine. I will try OTA later and report back. The only issue left is GPU acceleration. I think this should be pretty easy.

If you guys want to try the EFI, check out the 13.0 branch: [Link](https://github.com/SkyrilHD/Dell-E7250-Hackintosh/tree/13.0)

![Screenshot 2022-10-31 at 02 28 10](https://user-images.githubusercontent.com/28839925/198916489-cf46b74f-e71f-4204-be48-7f36ed598974.png)

**Reply by SkyrilHD** [reply]
And here we go, E7250 (and other Latitude Exx50 models) are ready for Ventura!

![Screenshot 2022-10-31 at 03 38 44](https://user-images.githubusercontent.com/28839925/198919984-5ba0e437-6ba3-4696-9f17-171519820af3.png)

![Screenshot 2022-10-31 at 03 37 58](https://user-images.githubusercontent.com/28839925/198919996-68025500-d6ec-429c-9ccc-4180a9cad8e3.png)

**Comment by SkyrilHD** [comment]
v4.0 has been released with official Ventura support!

https://github.com/SkyrilHD/Dell-E7250-Hackintosh/releases/tag/v4.0
